### ISSUE TITLE
Certain Mario Party 1, 2 and 3 Minigames run slower then they do on real hardware.

### ISSUE THREAD

**Issue by HowManyNames** [simple64/simple64#406]
**Describe the bug**
When playing certain mini-games in Mario Party 1, 2, and 3, the frame rate of the game will be noticeably lower then it would be on real hardware. I've been led to understand that this is usually due to a CPU bottleneck, but given the hardware I have my doubts. 

**To Reproduce**
1. Boot up Mario Party 3
2. Head over to the Mini-Game room (Yellow Star)
3. Talk to Toad, head over to the blue books (4 Player mini-games), play any mini-game that has to render 4 individual screens, some good examples for Mario Party 3 are Aces High, Frigid Bridges, and Awful Tower.
4. Play the mini-game, you'll notice the lag immediately.

**Expected behavior**
Given the accuracy of the emulator otherwise, I would expect the emulator to run the game at a frame-rate similar to that of real hardware

**Screenshots**
How the game should run: https://youtu.be/Knfez1Zg9BQ
How the game actually runs: https://streamable.com/dxvi5u

**Desktop (please complete the following information):**
 - OS: Windows
 - Version: 10
 - GPU: NVIDIA RTX 3070
 - CPU: AMD RYZEN 5800X3D

**Additional context**
A Raphnet N64 adapter N64 adapter and its corresponding plug-in was used to test the game on Simple64, however, the same behavior can be observed while using the normal controller plug-in, and a normal game-pad. Here's a save file to help out with testing.
https://github.com/simple64/simple64/files/13398683/Mario.Party.3.U.-76A8BBC8.zip

**Comment by CoderTimZ** [comment]
I'm experiencing this issue as well, and I know it's not an issue with my hardware being too slow because the emulator is able to run at 4x the speed during these mini-games if I toggle the speed limiter off.

**Comment by ksivaneri001** [comment]
I can say I'm experiencing this issue as well. Other examples I've noticed are Hyper Hydrants from MP3 and Shell Shocked from MP2.

**Comment by shazyv** [comment]
Yeah I can say this is something that affects all three games. It's mainly the games where it's emulating 4 screens at once most of the time but not always.

**Comment by austblack88** [comment]
I am also experiencing this issue. Like @shazyv said, I notice it especially on minigames with splitscreen.
